website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -96,62 +101,69 @@ We strongly recommend providing visible chevrons on most instances of ToggleIcon
 
 ### Placement
 
-In the event that the Toggle is positioned on the left side of the screen, the list can be aligned to the left side to fit more appropriately within the UI.
+Lists can be positioned to the left or right of the Toggle, and above or below the Toggle to fit more appropriately within the UI. Lists do not currently have collision detection. 
 
-![Dropdown list placement examples](/assets/components/dropdown/dropdown-placement.png =467x*)
+![Dropdown list placement examples](/assets/components/dropdown/dropdown-placement.png =963x*)
 
 ### Size
 
 #### Default width
 
 Lists have a minimum width of 200px and a maximum width of 400px. This means if there’s a long string in a ListItem the List will automatically expand up to 400px to accommodate that content before the content wraps.
 
-<Doc::ListContainer class="hds-dropdown-list">
-  <Hds::Dropdown::ListItem::Title @text="Signed in as" />
-  <Hds::Dropdown::ListItem::Description @text="name@email.com" />
-  <Hds::Dropdown::ListItem::Separator />
-  <Hds::Dropdown::ListItem::Interactive @text="User settings" />
-  <Hds::Dropdown::ListItem::Interactive @text="Admin" />
-  <Hds::Dropdown::ListItem::Interactive @text="Sign out" />
-</Doc::ListContainer>
+<!-- TODO: figure out how to fix the padding -->
+<div class="hds-dropdown__content">
+  <ul class="hds-dropdown__list">
+    <Hds::Dropdown::ListItem::Title @text="Signed in as" />
+    <Hds::Dropdown::ListItem::Description @text="name@email.com" />
+    <Hds::Dropdown::ListItem::Separator />
+    <Hds::Dropdown::ListItem::Interactive @text="User settings" />
+    <Hds::Dropdown::ListItem::Interactive @text="Admin" />
+    <Hds::Dropdown::ListItem::Interactive @text="Sign out" />
+  </ul>
+</div>
```

Comment:
@alex-ju I tried updating some examples, but I believe I'm doing something wrong because they don't look quite right. Can you help point me in the right direction?

Reply:
Oh, that's unexpected. What happens is that the docs website automatically adds a `doc-markdown-ul` class to all list elements, including these ones. So we're dealing with some style overrides. I'll try to find a solution to this.

Reply:
This is resolved now – see my last comment below.